app/views/layouts/application.html.erb

```diff
@@ -30,14 +31,17 @@
   <% end %>
 
     <% banner_message = capture do %>
-      This is a new service - your <a href="">feedback</a> will help us to improve it.
+      <a href="">Send us feedback</a>
```

Comment:
Should this be a link without an actual target?

Reply:
For now, yes. We haven't decided where to link to yet, it's on the product radar.